### DISCUSSION TITLE
What happened to the example code?

### DISCUSSION THREAD

**Discussion by deepskydetail** [rstudio/keras#1415] [Q&A] [answered]
Hi!

I've been using https://blogs.rstudio.com/ai/posts/2018-09-20-eager-pix2pix/ to help guide me create a neural network. There used to be the full code linked. But it is no longer there. I was wondering where it went and will it ever be back?

Thanks!

**Comment by t-kalinowski** [comment, marked as answer]
Welcome! Thanks for asking.

Many of the examples still need to be updated after the transition from TF v1 to Tf v2. 
You can still view them in the keras "r2" branch: https://github.com/rstudio/keras/tree/r2/vignettes/examples.

We're in the process of add up-to-date examples to https://keras.posit.co/articles/examples/. 

Contributions are welcome! To add an example, please open a PR adding a Rmd to the 'vignettes-src' folder: https://github.com/rstudio/keras/tree/main/vignettes-src. 

Examples can be adapted to R versions of those found at https://keras.io/examples/ (https://github.com/keras-team/keras-io/tree/master/examples), updated "old" R examples, or completely new examples.

**Comment by deepskydetail** [comment]
Thanks for the information!
